metadata_mapper/mappers/oai/omeka/nothumb_mapper.py

```diff
@@ -0,0 +1,18 @@
+from ..omeka_mapper import OmekaRecord, OmekaVernacular
+
+
+class NothumbRecord(OmekaRecord):
+    pass
+
+
+class NothumbVernacular(OmekaVernacular):
+    record_cls = NothumbRecord
+
+    def skip(self, record):
+        """
+        This is a lightweight version of OmekaRecord.map_is_shown_by(). Any changes here may need to be
+        reflected there.
+        """
```

Comment:
Thanks for the code comment. It's not great, but I agree with your decision to leave it like this for now. If this becomes a regular issue, then maybe we should re-think how we handle skips.